test/parallel/test-cluster-shared-handle-bind-privileged-port.js

```diff
@@ -35,6 +36,20 @@ if (common.isWindows)
 if (process.getuid() === 0)
   common.skip('as this test should not be run as `root`');
 
+// Some systems won't have port 42 set as a privileged port, in that
+// case, skip the test.
+if (common.isLinux) {
+  try {
+    const unprivilegedPortStart = parseInt(readFileSync('/proc/sys/net/ipv4/ip_unprivileged_port_start'));
+    if (unprivilegedPortStart <= 42) {
+      common.skip('Port 42 is unprivileged');
```

Comment:
```suggestion
      common.skip(`Port ${unprivilegedPortStart} is unprivileged`);
```

Reply:
I disagree, this is how it is setup in other places, and we are using port 42 throughout the test.

Reply:
Because, on some machines, that port number is unprivileged. This check ensures that 42 is a privileged port.

Reply:
But if `unprivilegedPortStart` is `41` we are skipping the test with an incorrect message.

Reply:
I disagree. If the unprivileged port starts at 41, port 42 is still unprivileged.

Reply:
I disagree. If it starts at 41, than 0-41 are privileged, while 42 is unprivileged, right?

Reply:
If you insist, I can modify the error message / skip case to be more specific

Reply:
I see what you mean. I have no strong opinion.

Reply:
FWIW it seems (albeit mildly) useful to know which are the priviledged ports in the skip message in case someone's debugging